### ISSUE TITLE
label consistency regularization for unlabeled data cifar100

### ISSUE THREAD

**Issue by cxxPulsar** [lijichang/LNL-NCE#3]
Hi,
It would be great if you can tell me why loss of unlabeled data should be calculated with 'cat[inputs_us, inputs_xs]'  in cifar100,but not in other dataset such as cifar10.I didn't find the explanation for that in paper.
Thanks!
The part of the code is:

        # label consistency regularization for unlabeled data
        if (args.dataset == 'cifar100') and ((args.r==0.2) or (args.r==0.5)):
            all_inputs, all_labels, all_masks = torch.cat([inputs_us, inputs_xs], dim=0), torch.cat([labels_u,  labels_x], dim=0), torch.cat([mask_u, mask_x], dim=0)
            all_logits = net(all_inputs)
            Lu = (F.cross_entropy(all_logits, all_labels, reduction='none') * all_masks.float()).mean()
        else:
            logits_us = net(inputs_us)
            Lu = (F.cross_entropy(logits_us, labels_u, reduction='none') * mask_u.float()).mean()

**Comment by lijichang** [comment]
Dear,

Thank you for your interest in our work. 

To some extent, the identified clean samples and the relabeled noisy samples serve similar purpose. If you want to achieve better performance on CIFAR-10, you can also use the same training strategy as CIFAR-100 to train the model on CIFAR-10.

Best.

**Comment by cxxPulsar** [comment]
Thanks for your response! By the way,could you tell me why isn't unlabeled data participate in mixup？

**Comment by lijichang** [comment]
> Thanks for your response! By the way,could you tell me why isn't unlabeled data participate in mixup？

MixUp is a supervised regularization technique that requires samples‘  labels.

**Comment by lijichang** [comment]
> Thanks for your response! By the way,could you tell me why isn't unlabeled data participate in mixup？

@cxxPulsar 

MixUp is a supervised regularization technique that requires samples‘  labels.
